pkg/cmd/env/connect.go

```diff
@@ -0,0 +1,117 @@
+package env
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	"github.com/coreeng/corectl/pkg/cmdutil/userio"
+	corectlenv "github.com/coreeng/corectl/pkg/env"
+	"github.com/coreeng/corectl/pkg/gcp"
+	"github.com/coreeng/developer-platform/pkg/environment"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var localPort int
+
+func connectCmd(cfg *config.Config) *cobra.Command {
+	connectCmd := &cobra.Command{
+		Use:   "connect [environment]",
+		Short: "Connect to an environment",
+		Long:  `This command allows you to connect to a specified environment.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			io := userio.NewIOStreams(
+				cmd.InOrStdin(),
+				cmd.OutOrStdout(),
+			)
+			return connect(cfg, io)
+		},
+	}
+
+	connectCmd.Flags().IntVar(&localPort, "localPort", 54808, "The local port to use for connection to the cluster")
```

Comment:
I would propose to create an explicit structure with options required for the command to avoid global variables.
Example: https://github.com/coreeng/corectl/blob/980ff32f335e82aa369acbb357c1772a0ecd9daa/pkg/cmd/application/create/app_create.go#L21